tests/system/Helpers/TextHelperTest.php

```diff
@@ -348,4 +348,15 @@ public function testEmptyAlternator()
 
         $this->assertSame(' scream!  scream!  scream!  scream! ', $result);
     }
+
+    public function testStrContains()
+    {
+        $needle = 'codeigniter';
+        $string = 'codeigniter is small but powerful php framework';
+        $this->assertStringContainsString($needle, $string);
```

Comment:
This is a PHPUnit method and does not cover your newly added function. In here, it seems you're testing the PHPUnit method.